src/pages/homepage/RisingSection/RisingList.tsx

```diff
@@ -0,0 +1,104 @@
+/** @jsxImportSource @emotion/react */
+import type { Product } from "@/types/api_types";
+import styled from "@emotion/styled";
+import { useState } from "react";
+import { useNavigate, useSearchParams } from "react-router-dom";
+import { useSuspenseApiQuery } from "@/hooks/useSuspenseApiQuery";
+import RisingItem from "@/pages/homepage/RisingSection/RisingItem";
+import { API_ENDPOINTS } from "@/utils/API_ENDPOINTS";
+
+const INITIAL_VISIBLE_COUNT = 6;
+
+export default function RisingList() {
+  const [visibleCount, setVisibleCount] = useState(INITIAL_VISIBLE_COUNT);
+  const navigate = useNavigate();
+  const [searchParams] = useSearchParams();
+  const targetType = searchParams.get("targetType") ?? "ALL";
+  const rankType = searchParams.get("rankType") ?? "MANY_WISH";
+
+  const { data } = useSuspenseApiQuery<Product[]>({
+    queryKey: [API_ENDPOINTS.RANKING, targetType, rankType],
+    url: API_ENDPOINTS.RANKING,
+    params: { targetType, rankType },
+  });
```

Comment:
```suggestion
  const { data } = useProducts();
```

대놓고 SuspenseApiQuery를 드러내기보다 한 단계 추상화해서 사용해주면 어떨까요?
가령, 지금 ReqctQuery로 적용하는 과정을 거쳤는데
추상화 수준이 잘 격리되어있으면 컴포넌트쪽 레이어를 건드리지 않고 훅만 수정해서 어플리케이션이 동작하도록 만들 수도 있답니다 ㅎㅎ

useProducts() 라는 훅의 내부 구현을 감추고 필요한것만 노출시키는거죠!

Reply:
SuspenseApiQuery를 useProducts내에서 사용하도록 추상화 했습니다!